setup.py

```diff
@@ -2,33 +2,26 @@
 
 setup(
     name='libcovebods',
-    version='0.5.0',
+    version='0.6.0',
     author='Open Data Services',
     author_email='code@opendataservices.coop',
-    url='https://github.com/open-contracting/lib-cove-ocds',
+    url='https://github.com/openownership/lib-cove-bods',
     description='A data review library',
     packages=find_packages(),
     long_description='A data review library',
     install_requires=[
-        'jsonref',
-        'jsonschema<2.7',
-        # Required for jsonschema to validate URIs
-        'rfc3987',
-        # Required for jsonschema to validate date-time
-        'strict-rfc3339',
-        'CommonMark',
-        'Django',
-        'bleach',
-        'requests',
-        'json-merge-patch',
-        'cached-property',
         'python-dateutil',
-        # TODO Should also have flatten-tool >= v0.5.0 - that is currently in requirements instead.
-        # TODO Should also have lib-cove  >= v0.3.1 - that is currently in requirements instead.
+        'Django>1.11.23,<1.12',
+        'flattentool>=0.5.0',
+        'libcove>=0.6.0'
```

Comment:
I went to v0.13.0 here, because before that libcove didn't specify all its dependencies, that might be a clue to the specs?